public/modules/custom/grants_admin_applications/src/Form/ResendApplicationsForm.php

```diff
@@ -355,8 +346,8 @@ public static function getStatus(array $form, FormStateInterface $formState): vo
       }
     }
     catch (\Exception $e) {
-      $messenger->addError($e->getMessage());
-      $logger->error(
+      $this->messenger()->addError($e->getMessage());
```

Comment:
Printing exception message might expose some sensitive information.

Reply:
This can imo be printed, it's for admin users.